kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -99,15 +104,16 @@
         </template>
       </UiToolbar>
     </header>
-    <!-- IF making changes to the sub nav, make sure to make -->
-    <!-- corresponding changes in SideNav.vue in regards to  -->
-    <!-- Window size and app context. Changes may need to be made -->
-    <!-- in parallel in both files for non-breaking updates -->
```

Comment:
yay that this convoluted thing isn't needed anymore!

Reply:
This was mostly achieved by doing some prop-drilling instead, so that the components are controlled by the same flag at the top level in the AppBarPage.